sdk/resourcemanager/Azure.ResourceManager.Core/tests/Unit/TestAssets/Identity/SystemAssigned.json

```diff
@@ -1,8 +1,8 @@
 ﻿{
-  "identity": {
-    "principalId": "22fddec1-8b9f-49dc-bd72-ddaf8f215577",
-    "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-    "type": "SystemAssigned",
-    "userAssignedIdentities": null
-    }  
+    "identity": {
```

Comment:
this isn't how the service is going to return it.  It will come back with principalid and tenantid for the system under the identity